xlators/features/quiesce/src/quiesce.c

```diff
@@ -130,8 +127,7 @@ gf_quiesce_failover_cbk(call_frame_t *frame, void *cookie, xlator_t *this,
          * just abort the failover attempts without retrying with other
          * hosts.
          */
-        gf_msg(this->name, GF_LOG_INFO, op_errno, QUIESCE_MSG_FAILOVER_FAILED,
-               "Initiating failover to host:%s failed:", (char *)cookie);
+        GF_LOG_I(this, QUIESCE_MSG_FAILOVER_FAILED(cookie, op_errno));
```

Comment:
Nice! This will make people to 'Define' error log. Looks clean IMO.

Reply:
Yes i also believe it is clean and easy to use.